config/checkstyle-input-suppressions.xml

```diff
@@ -12,6 +12,12 @@
             files="[\\/]checks[\\/]coding[\\/]illegaltype[\\/]InputIllegalTypeSameFileNameFalsePositive.java"/>
   <suppress id="noCheckstyleInInputs"
             files="[\\/]checks[\\/]coding[\\/]illegaltype[\\/]InputIllegalTypeTestSameFileNameGeneral.java"/>
+  <suppress id="noCheckstyleInInputs"
+            files="[\\/]checks[\\/]metrics[\\/]classfanoutcomplexity[\\/]InputClassFanOutComplexityExcludedPackagesAllIgnored.java"/>
+  <suppress id="noCheckstyleInInputs"
+            files="[\\/]checks[\\/]metrics[\\/]classfanoutcomplexity[\\/]InputClassFanOutComplexityExcludedPackagesCommonPackage.java"/>
+  <suppress id="noCheckstyleInInputs"
+            files="[\\/]checks[\\/]metrics[\\/]classfanoutcomplexity[\\/]InputClassFanOutComplexityExcludedPackagesDirectPackages.java"/>
```

Comment:
These must be removed from below. We should see a symmetrical diff in this file; otherwise we will have duplicate suppressions.